### DISCUSSION TITLE
arm64 build for docker pull ghcr.io/open-telemetry/opentelemetry-java-instrumentation/smoke-test-fake-backend:20220510.2302010706

### DISCUSSION THREAD

**Discussion by indira-traceable** [open-telemetry/opentelemetry-java-instrumentation#6028] [Q&A]
do we have the arm64 for the mock collector   `ghcr.io/open-telemetry/opentelemetry-java-instrumentation/smoke-test-fake-backend:20220510.2302010706`
I get the warning when it is started on arm64/centos 
```
sudo docker run -d -p 8080:8080 ghcr.io/open-telemetry/java-test-containers:smoke-fake-backend-20210324.684269693
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
```
please let me know, thanks in advance

**Comment by trask** [comment]
hi @indira-traceable! I don't think GitHub Actions supports ARM (yet), so we haven't had a need for ARM images, but if you would like to send a PR, it would be gladly accepted

**Reply by anuraaga** [reply]
As a reference, a snippet that would allow building on both is this one

https://github.com/aws-observability/aws-otel-java-instrumentation/blob/main/sample-apps/springboot/build.gradle.kts#L23
